<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Consumer</string>
  </property>
  <widget class="QWidget" name="allFather">
   <property name="minimumSize">
    <size>
     <width>775</width>
     <height>444</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="serverIpBox">
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>101</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>291</width>
        <height>101</height>
       </size>
      </property>
      <property name="title">
       <string>Server IP</string>
      </property>
      <widget class="QLineEdit" name="serverIp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>271</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="connectBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="disconnectBtn">
       <property name="geometry">
        <rect>
         <x>148</x>
         <y>60</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
      <widget class="QLabel" name="stateLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>0</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;i&gt;&lt;font color=&quot;red&quot;&gt;Disconnected&lt;/font&gt;&lt;/i&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="listBox">
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>191</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>291</width>
        <height>191</height>
       </size>
      </property>
      <property name="title">
       <string>Data Producers</string>
      </property>
      <widget class="QListWidget" name="producersList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>271</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="updateBtn">
       <property name="geometry">
        <rect>
         <x>188</x>
         <y>160</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Collecting data from:</string>
       </property>
      </widget>
      <widget class="QLabel" name="conectedToLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="timingBox">
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>121</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>291</width>
        <height>121</height>
       </size>
      </property>
      <property name="title">
       <string>Time Controller</string>
      </property>
      <widget class="QLabel" name="timingLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Timing(s)</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="timingLcd">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>50</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
      <widget class="QSlider" name="timingSlider">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>50</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="startBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stopBtn">
       <property name="geometry">
        <rect>
         <x>158</x>
         <y>90</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QLabel" name="startedLabel">
       <property name="geometry">
        <rect>
         <x>186</x>
         <y>0</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;i&gt;&lt;font color=&quot;red&quot;&gt;Not started yet&lt;/font&gt;&lt;/i&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QGroupBox" name="plotBox">
      <property name="minimumSize">
       <size>
        <width>460</width>
        <height>425</height>
       </size>
      </property>
      <property name="title">
       <string>Data Plot</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="Plotter" name="plot" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>timingSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>timingLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
